src/shared/sandboxCommandBase.ts

```diff
@@ -150,6 +150,27 @@ export abstract class SandboxCommandBase<T> extends SfCommand<T> {
       this.updateProgress(results, options.isAsync);
       this.reportResults(results);
     });
+
+    lifecycle.on(SandboxEvents.EVENT_MULTIPLE_SBX_PROCESSES, async (results: SandboxProcessObject[]) => {
+      const resumingProcess = results.shift() as SandboxProcessObject;
```

Comment:
avoid mutating the array, especially one passed into a function.  `Array.shift` mutates the array, and whoever emitted the event might be using the array or passing it to someone else who expects the array[0] to still be present.

Reply:
`const [resumingProcess, ...others] = results` would get you something similar to what shift was doing

Reply:
I did the shift deliberately but it's a good point about other listeners being affected.  I'll change it.